content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -102,13 +102,21 @@ to ensure that the API presents a clear, consistent view of system resources
 and behavior, and to enable controlling access to end-of-life and/or
 experimental APIs.
 
-Refer to [API versions reference](/docs/reference/using-api/api-overview/#api-versioning)
-for more details on the API version level definitions.
-
 To make it easier to evolve and to extend its API, Kubernetes implements
 [API groups](/docs/reference/using-api/api-overview/#api-groups) that can be
 [enabled or disabled](/docs/reference/using-api/api-overview/#enabling-or-disabling).
 
+API resources are distinguished from each other based on their API groups and
```

Comment:
If you have time, would you fix line 71, the table caption:
`
 <caption>Valid request header values for OpenAPI v2 queries</caption>
`
This line is showing up after the table (in light gray).
The table shortcode could be used instead.

Reply:
I opened issue #24413 .

Reply:
Investigated the HTML differences. It looks like the `table caption` shortcode would render the caption as `display:none`. I'll do that as well.
I did not change the table from HTML to markdown because raw HTML is used in this scenario for a good reason -- there are row spans that are not supported by markdown.
I'll go ahead and close #24413.